cmd/repo_set_default.go

```diff
@@ -25,8 +25,12 @@ func newRepoDefaultCmd(config *RootCommandConfig) *cobra.Command {
 	// initCmd represents the init command
 	var setDefaultCmd = &cobra.Command{
 		Use:   "set-default <name>",
-		Short: "Set desired default repository",
-		Long:  ``,
+		Short: "Sets a default repository.",
+		Long: `Sets your specified repository to be the default repository. The "incubator" repository is the initial default repository you will get upon installing Appsody. The default repository cannot be removed from you list of configured Appsody repositories. To remove a default repository, you would first have to set a different default repository.
```

Comment:
perhaps this can be deleted from this command.. The "incubator" repository is the initial default repository you will get upon installing Appsody. The default repository cannot be removed from you list of configured Appsody repositories. To remove a default repository, you would first have to set a different default repository.